src/random-seed/random-seed.c

```diff
@@ -242,6 +245,13 @@ static int run(int argc, char *argv[]) {
                         if (r < 0)
                                 log_error_errno(r, "Failed to write seed to /dev/urandom: %m");
                 }
+                if (write_seed_file) {
```

Comment:
Please add a comment in the code too, so that we don't have to go back to the commit message to find out why this is being done. A short one is fine, just to give a hint.

Reply:
Will do.